src/features/navigation/RootNavigator/RootNavigator.tsx

```diff
@@ -26,6 +26,7 @@ import { Categories as SearchCategories } from 'features/search/pages/Categories
 import { LocationFilter } from 'features/search/pages/LocationFilter'
 import { LocationPicker } from 'features/search/pages/LocationPicker'
 import { SearchFilter } from 'features/search/pages/SearchFilter'
+import { FirstTutorial } from 'features/tutorials/pages/FirstTutorial'
```

Comment:
perso, j'ai mis le composant `CulturalSurvey` dans `features/firstLogin` .
Il faut qu'on décide si on : 
- garde uniquement firstLogin
- garde uniquement tutorials
- autre option si il y a lieu
```suggestion
import { FirstTutorial } from 'features/firstLogin/pages/FirstTutorial'
```

Reply:
Bonne remarque !
Les deux ont du sens, je ne sais pas si j'ai un argument plus en faveur de l'un ou de l'autre